### PR TITLE
Require Node.js 10 and bug fixes

### LINKED ISSUE #18: Calling the cli with no git email results in `undefined` being inserted
The `undefined` is super hard to spot. Maybe a better placeholder like `<EMAIL>` could be inserted in the case that the git call doesn't return anything useful.

### PR DESCRIPTION
**Pull request by brandon93s**
Apologies for the large PR -- this contains broad updates for the module.  Please let me know if you'd prefer to see this broken up into smaller branches. Updates included:

1. Update existing and add new translations [[source](https://www.contributor-covenant.org/translations/)]
2. Fix the email token - `[INSERT EMAIL ADDRESS]` - across all translations
3. Fix #18 ( details in comments )
4. Upgrade all dependencies
5. Drop node 8 support

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -101,15 +101,59 @@ function loadLanguages() {
...
 	let email;
 	try {
 		email = execa.sync('git', ['config', 'user.email']).stdout.trim();
-	} catch (_) {}
+	} catch {}
 
 	return email;
 }
 
+async function findEmail(existingSrc) {
```
Comment (brandon93s) [inline cli.js:113]: Refactors out email resolution to be consistent between the _new_ and _upgrade_ codepaths.  There are a few scenarios - e.g. upgrading an existing Code of Conduct that doesn't contain an email - that will result in the `undefined` scenario from #18.  This should force the prompt/error in all paths.

File: cli.js [hunk 2]
```diff
@@ -128,56 +172,36 @@ function generate(filepath, email) {
 }
 
 async function init() {
+	const directoryPathParts = flags.directory.split(path.sep);
```
Comment (brandon93s) [inline cli.js]: Fixing a bug when passing in a Windows path via `--directory`.

File: vendor/code-of-conduct.bn.md [hunk 3]
```diff
@@ -1,67 +1,63 @@
...
+কোনো অনলাইন বা অফলাইন অনুষ্ঠানে নিযুক্ত প্রতিনিধি হিসেবে কাজ করা। প্রজেক্ট মেনটেইনাররা প্রতিনিধিত্ব-এর
 সংজ্ঞা আরও ভাল ভাবে ব্যাখ্যা এবং স্পষ্ট করতে পারে।
 
 
 ## প্রয়োগ
 
-অপমানজনক, হয়রানী বা গ্রহণযোগ্য নয় এমন আচরণের খবর প্রজেক্ট টীমকে জানানো যাবে [এই ই-মেল এ]। 
-সব অভিযোগ পর্যালোচনা করা ও তদন্ত করা হবে এবং এই পরিস্থিতিতে যে প্রতিক্রিয়া প্রয়োজনীয় ও উপযুক্ত 
-বলে মনে করা হবে তা প্রয়োগ করা হবে। প্রজেক্ট টীম এই ঘটনার রিপোর্টার সম্পর্কে গোপনীয়তা বজায় রাখতে বাধ্য। 
+অপমানজনক, হয়রানী বা গ্রহণযোগ্য নয় এমন আচরণের খবর প্রজেক্ট টীমকে জানানো যাবে [INSERT EMAIL ADDRESS]।
```
Comment (brandon93s) [inline vendor/code-of-conduct.bn.md:55]: Bug from my original implementation in #13, these placeholders wouldn't be swapped out.

File: cli.js (hunk 2 above)
Comment (brandon93s) [reply cli.js]: Could alternatively pull in `upath` or `slash`.

**Comment by sindresorhus** [comment]
Perfect 👌

**Pull request merged**